x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/simple_privilege_section/simple_privilege_section.tsx

```diff
@@ -226,6 +226,7 @@ export class SimplePrivilegeSection extends Component<Props, State> {
                   privilegeIndex={this.props.role.kibana.findIndex((k) =>
                     isGlobalPrivilegeDefinition(k)
                   )}
+                  canAccessRoleManagement={true}
```

Comment:
shouldn't this reflect the privileges user has, i.e `manage_security` or `read_security`?

Reply:
@elena-shostak This flag only controls when the "Additional Stack Management permissions can be found outside of this menu, in index and cluster privileges." message is shown.

Maybe I just call it something like, `showAdditionalPermissionsMessage` ?